### PR TITLE
Post message parameters

### PR DESCRIPTION
**Pull request by Sayuris1**

### CODE REVIEW

File: in/cursor.script [hunk 1]
```diff
@@ -91,7 +91,7 @@ local function notify_event(self, game_object_id, game_object_group, action, mes
 		msg.post(".", message_id, message)
 	end
 	if game_object_id and exists then
-		msg.post(game_object_id, message_id)
+		msg.post(game_object_id, message_id, message)
```
Comment (britzl) [inline in/cursor.script]: The message table only gets updated when self.notify_own_gameobject is true. I suggest that we update the message table before the self.notify_own_gameobject check.

File: in/cursor.script [hunk 2]
```diff
@@ -83,15 +83,15 @@ local function notify_event(self, game_object_id, game_object_group, action, mes
...
 	if game_object_id and exists then
-		msg.post(game_object_id, message_id, message)
+			message.id = game_object_id
+			message.group = game_object_group
+			message.x = action.x
+			message.y = action.y
+			msg.post(game_object_id, message_id, message)
+		if self.notify_own_gameobject then
+			msg.post(".", message_id, message)
+		end
```
Comment (Sayuris1) [inline in/cursor.script]: Is there any reason we notify notify own game object if game_object_id and exists check fails ?

Comment (britzl) [reply in/cursor.script]: The indentation on that file is strange. But in the snippet of code above we do not notify own game object if the other game object does not exist right?

File: in/cursor.script [hunk 3]
```diff
@@ -83,15 +83,15 @@ local function notify_event(self, game_object_id, game_object_group, action, mes
...
-		msg.post(game_object_id, message_id)
+			message.id = game_object_id
+			message.group = game_object_group
+			message.x = action.x
+			message.y = action.y
+			msg.post(game_object_id, message_id, message)
+		if self.notify_own_gameobject then
+			msg.post(".", message_id, message)
+		end
 	end
```
[the remark this answers is not in the document]
Comment (britzl) [reply in/cursor.script]: Ok. can you please fix the indentation and also make sure the own game object always gets notified of events?

**Comment by britzl** [comment]
Fixed in a recent release. Closing.

**Pull request closed without merging**